docs/building/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 section-break: true
-nav-title: '<i class="fa fa-cogs title maindish" aria-hidden="true"></i> Project Setup'
-title: "Project Setup"
-nav-id: "start"
+nav-title: '<i class="fa fa-cogs title dessert" aria-hidden="true"></i> Building & Developing Flink'
```

Comment:
Yeah, I had that first but decided to change it because users (who don't want to develop Flink) might want to build a custom Flink version for a specific Hadoop version or to include a patch.

Therefore, I chose "Buiding & Developing Flink".

Reply:
Building Flink from source, as you said, may be a demand of users who don't want to develop it. 
With content in your version, this section would be at quite bottom.
And since choose specific version of Flink, Hadoop, ... is a common demand of users, also according to `building/building.html`'s content, I'd prefer move it to `Tutorials`, and link to it from `Developing Flink`.

Reply:
Sorry if you see the previous comment I deleted. 

The only thing remain from that is, as you said, choose different version of Flink or Hadoop might be a command demand from users. Even they don't want to develop Flink. So it would be worth to link the building page from `tutorials/local_setup.html`'s `Download and Compile` section.  Mention that one could config the version as he like.

Reply:
Hmm, I see what you mean. I think the build instructions are useful in two cases.

* Developing Flink
* Deployment and Operations.

Maybe we just cross-link the page

Reply:
That would be perfect